css-gaps-1/Overview.bs

```diff
@@ -0,0 +1,601 @@
+<pre class='metadata'>
+Title: CSS Gap Decorations Module Level 1
+Shortname: css-gaps
+Level: 1
+Status: ED
+Group: csswg
+Work Status: exploring
+Repository: w3c/csswg-drafts
+URL: https://drafts.csswg.org/css-gaps-1
+Editor: Kevin Babbitt, Microsoft, https://github.com/kbabbitt, w3cid 124689
+Abstract: This module introduces several properties to add row and column gap decorations to container layout types such as grid and flex.
+</pre>
+<pre class='anchors'>
+urlPrefix: https://drafts.csswg.org/css-grid-3/; type: dfn; spec: CSS-GRID-3; text: masonry container
+</pre>
+
+Introduction {#intro}
+=====================
+
+	<em>This section is not normative.</em>
+
+	[[CSS-MULTICOL-1#column-gaps-and-rules]] allows for rules to be drawn
+	between columns in a <a>multicol container</a>. This specification expands
+	upon the 'column-rule-width', 'column-rule-style', and 'column-rule-color'
+	properties, adding equivalents in the row direction, expanding their
+	application to other container layouts, and giving advanced control over
+	where and how gap decorations are painted.
+
+Value Definitions {#values}
+---------------------------
+
+	This specification follows the <a href="https://www.w3.org/TR/CSS2/about.html#property-defs">CSS property definition conventions</a> from [[!CSS2]]
+	using the <a href="https://www.w3.org/TR/css-values-3/#value-defs">value definition syntax</a> from [[!CSS-VALUES-3]].
+	Value types not defined in this specification are defined in CSS Values &amp; Units [[!CSS-VALUES-3]].
+	Combination with other CSS modules may expand the definitions of these value types.
+
+	In addition to the property-specific values listed in their definitions,
+	all properties defined in this specification
+	also accept the <a>CSS-wide keywords</a> as their property value.
+	For readability they have not been repeated explicitly.
+
+Gap decorations {#gap-decorations}
+==================================
+
+	Various layouts in CSS such as
+	<a>multicol containers</a>, <a>flex containers</a>, <a>grid containers</a>, and <a>masonry containers</a>
+	position child boxes adjacent to each other with gaps, also known as <a>gutters</a>, between them.
+	Each such gap may contain a <dfn>gap decoration</dfn> which authors
+	may use to provide a visual separator between adjacent boxes.
```

Comment:
Is "visual separator" the right terminology if `gap` is defining the separation. Maybe something that notes it is used to style the gaps instead of as the separator itself?

If so, it might make sense to have a link to the definition of `gap` here (but maybe that wouldn't make sense since I don't think multicol has a `gap` property)

Reply:
> Is "visual separator" the right terminology if gap is defining the separation. Maybe something that notes it is used to style the gaps instead of as the separator itself?

Re-worded this a bit; let me know what you think.

> If so, it might make sense to have a link to the definition of gap here (but maybe that wouldn't make sense since I don't think multicol has a gap property)

The "gutter" link is intended to serve this purpose - this is the term defined in css-align-3 and used in the multicol and grid specs.